### PR TITLE
Less bad

### PR DESCRIPTION
**Pull request by markstory**

### CODE REVIEW

File: somefile.py [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 log = logging.getLogger(__name__)
 
+def funcName(stuff):
```
Comment (markstory) [inline somefile.py]: E302 expected 2 blank lines, found 1

File: somefile.py [hunk 2]
```diff
@@ -4,6 +4,12 @@
...
+  if (thing):
```
Comment (markstory) [inline somefile.py]: E111 indentation is not a multiple of four

File: somefile.py [hunk 3]
```diff
@@ -4,6 +4,12 @@
...
+   return True
```
Comment (markstory) [inline somefile.py]: E111 indentation is not a multiple of four

File: somefile.py [hunk 4]
```diff
@@ -4,6 +4,12 @@
...
+  else:
```
Comment (markstory) [inline somefile.py]: E111 indentation is not a multiple of four

**Comment by markstory** [comment]
The following commits had issues. The pattern '(?:[Ii]ssues?|[Ss]ees?|[Rr]ef(?:erences?|s?)|[Ff]ix(?:es)?|[Cc]loses?)\s+(?:#\d+|[a-zA-Z]+\-\d+) was not found in:
* 04ca9f623f929494a8c8c6591704d177fcfd0ee4

File: somefile.py (hunk 1 above)
Comment (markstory) [inline somefile.py]: E302 expected 2 blank lines, found 1

**Comment by markstory** [comment]
Looks grrreeaat

File: somefile.py [hunk 5]
```diff
@@ -5,6 +5,13 @@
 log = logging.getLogger(__name__)
 
 
+def funcName(stuff):
+    if thing:
```
Comment (stickler-ci-dev) [inline somefile.py]: F821 undefined name 'thing'

**Pull request closed without merging**